CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Version 1.2.1 (19JUL22)
+
+* **Monitoring**
+  * [CHANGE] - change_grafana_admin_password script has been created to change the Grafana admin password.
```

Comment:
This is a new FEATURE, not just a change...so use [FEATURE] rather than [CHANGE]